new-client/src/plugins/Edit/components/AttributeEditor.js

```diff
@@ -76,8 +76,15 @@ class AttributeEditor extends React.Component {
         } else {
           //If the feature has field: "" it will be changed to the default value.
           //Not sure if we want this behavior?
-          valueMap[field.name] =
-            featureProps[field.name] || field.defaultValue || "";
+          //Object that returns as a string results in [object] [Object]
+          if (
+            JSON.stringify(featureProps[field.name]) === '{"xsi:nil":"true"}'
```

Comment:
To be honest, I don't see why you compare a stringified object, instead of just
```featureProps[field.name]?.["xsi:nil"] === true```

Reply:
I'll write like this then:

featureProps[field.name]?.["xsi:nil"] === "true"
            ? (valueMap[field.name] = "")
            : (valueMap[field.name] =
                featureProps[field.name] || field.defaultValue || "");

Then it seems to work